mediapipe/tasks/ios/core/sources/MPPTaskRunner.mm

```diff
@@ -59,7 +62,20 @@ - (instancetype)initWithCalculatorGraphConfig:(CalculatorGraphConfig)graphConfig
 }
 
 - (BOOL)sendPacketMap:(const PacketMap &)packetMap error:(NSError **)error {
+  if (!_initializedWithPacketsCallback) {
+    [MPPCommonUtils
+        createCustomError:error
+                 withCode:MPPTasksErrorCodeFailedPreconditionError
+              description:[NSString
+                              stringWithFormat:@"This method can only be called if the task is "
+                                               @"running in a stream mode and an object of a class "
```

Comment:
"an object of a class" is a bit hard to parse. Maybe just "and a delegate is provided in the task options..."